### PR TITLE
LoggingAppender - removed DCL

### PR DESCRIPTION
**Pull request by saturnism**
Avoid using deprecated newInstance() method.
Removed double checked locking pattern - it doesn't work, and unnecessary, since start() is already synchronized.

### CODE REVIEW

**Comment by sduskis** [comment]
@saturnism, all of the builds fail.

**Comment by sduskis** [comment]
Can we remove the `getLogging()` method?

File: google-cloud-clients/google-cloud-contrib/google-cloud-logging-logback/src/main/java/com/google/cloud/logging/logback/LoggingAppender.java [hunk 1]
```diff
@@ -228,13 +239,6 @@ public synchronized void stop() {
   }
 
   Logging getLogging() {
-    if (logging == null) {
-      synchronized (this) {
-        if (logging == null) {
-          logging = getLoggingOptions().getService();
-        }
-      }
-    }
```
Comment (ajaaym) [inline google-cloud-clients/google-cloud-contrib/google-cloud-logging-logback/src/main/java/com/google/cloud/logging/logback/LoggingAppender.java]: where logging service would be created? Public constructor dont have logging parameter and other constructor with logging parameter is package private.

Comment (sduskis) [reply google-cloud-clients/google-cloud-contrib/google-cloud-logging-logback/src/main/java/com/google/cloud/logging/logback/LoggingAppender.java]: The change is to create the logging service in start.

File: google-cloud-clients/google-cloud-contrib/google-cloud-logging-logback/src/main/java/com/google/cloud/logging/logback/LoggingAppender.java [hunk 2]
```diff
@@ -190,18 +198,21 @@ public synchronized void start() {
     if (isStarted()) {
       return;
     }
+    super.start();
+
+    if (logging == null) {
+      logging = LoggingOptions.getDefaultInstance().getService();
```
Comment (ajaaym) [inline google-cloud-clients/google-cloud-contrib/google-cloud-logging-logback/src/main/java/com/google/cloud/logging/logback/LoggingAppender.java]: can you please use getLoggingOptions as it sets the credential if configured.

**Comment by sduskis** [comment]
@saturnism, can you please address the [issue](https://github.com/googleapis/google-cloud-java/pull/4756#discussion_r271746551) that @ajaaym raised?

**Pull request closed without merging**